graphql2/graphqlapp/rotation.go

```diff
@@ -375,3 +375,27 @@ func (m *Mutation) UpdateRotation(ctx context.Context, input graphql2.UpdateRota
 	}
 	return true, nil
 }
+
+func (a *Query) UpcomingHandoffTimes(ctx context.Context, input *graphql2.UpcomingHandoffTimesInput) ([]time.Time, error) {
+	var result []time.Time
+
+	if input.Hours <= 0 || input.Count <= 0 {
+		return result, errors.New("invalid upcoming handoff times input")
+	}
+
+	loc, err := time.LoadLocation(input.TimeZone)
```

Comment:
This error needs to be checked and handled, also use the util.LoadLocation helper, shouldn't have made it past `make check`